gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/experimental/uikit/internal/configureTaskToGenerateXcodeProject.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020-2022 JetBrains s.r.o. and respective authors and developers.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.compose.experimental.uikit.internal
+
+import org.gradle.api.Project
+import org.gradle.api.tasks.TaskProvider
+import org.jetbrains.compose.experimental.uikit.tasks.AbstractComposeIosTask
+
+internal fun Project.configureTaskToGenerateXcodeProject(
+    id: String,
+    projectName: String,
+    bundleIdPrefix: String,
+    teamId: String? = null,
+): TaskProvider<AbstractComposeIosTask> = tasks.composeIosTask<AbstractComposeIosTask>("iosGenerateXcodeProject$id") {
+    dependsOn(TASK_INSTALL_XCODE_GEN)
+    doLast {
+        val buildIosDir = getBuildIosDir(id)
+        buildIosDir.mkdirs()
+        buildIosDir.resolve("project.yml").writeText(
+            """
+            name: $projectName
+            options:
+              bundleIdPrefix: $bundleIdPrefix
+            settings:
+              ${if (teamId != null) "DEVELOPMENT_TEAM: \"$teamId\"" else ""}
+              CODE_SIGN_IDENTITY: "iPhone Developer"
```

Comment:
Is this always correct?

Reply:
The configuration is correct but not flexible.
In the future it will be necessary to take out as a separate parameter.